lib/main.coffee

```diff
@@ -11,6 +11,8 @@ Config = require './config'
 KernelManager = require './kernel-manager'
 Inspector = require './inspector'
 AutocompleteProvider = require './autocomplete-provider'
+zmq_services = require './zmq-services'
```

Comment:
We just refactor `KernelManager` into a class so that we don't have a singleton.
Please, don't make `ZMQServices` a singleton.